x-pack/plugins/observability/public/context/has_data_context.tsx

```diff
@@ -24,7 +24,7 @@ export type HasDataMap = Record<
   DataContextApps,
   {
     status: FETCH_STATUS;
-    hasData?: boolean | Alert[];
+    hasData?: boolean;
```

Comment:
The check on line 152 can be simplified now that `hasData` is always a `boolean`.

https://github.com/elastic/kibana/blob/e106bf3a3cf2ba3c446f6100d3f9b655a9be4256/x-pack/plugins/observability/public/context/has_data_context.tsx#L149-L154

Reply:
oh! I'm surprised `tsc` didn't get this. I'll fix it